backend/molgenis-emx2/src/main/java/org/molgenis/emx2/utils/generator/IdGenerator.java

```diff
@@ -0,0 +1,5 @@
+package org.molgenis.emx2.utils.generator;
+
+public interface IdGenerator {
+  public String generateId();
```

Comment:
why this interface? we will have only one id generator not mulptiple so then interfaces are not needed.

Reply:
to separate the api from the imp it makes the code more maintainable and testable